About.html

```diff
@@ -0,0 +1,128 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/about.css" />
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet">
+    <title>Imaginary Dance</title>
+  </head>
+  <body>
+    <div class="home-section">
+      <header>
+        <div class="top-header">
+          <ul class="top-header-list">
+            <li>
+              <a href=""><i class="fab fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-twitter"></i></a>
+            </li>
+            <li><a href="">English</a></li>
+            <li><a href="Index.html">My Page</a></li>
+            <li><a href="">Logout</a></li>
+          </ul>
+        </div>
+        <nav>
+          <div class="modal">
+            <img src="./Image/Enabled.svg" class="close" alt="closeTag" />
+            <div class="modal-body">
+              <ul class="modal-menu">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="About.html">About</a></li>
+                <li><a href="#">Feature</a></li>
+              </ul>
+            </div>
+          </div>
+            <div class="logo">
+                <img src="./image/logo.png" alt="logo">
+              </div>
+          <div class="nav-hamburger">
+            <img src="./image/button.png" class="hamburger" alt="" />
+          </div>
+          <div class="menu-icons">
+            <ul>
+              <li><a href="index.html">Home</a></li>
+              <li><a href="#">Feature</a></li>
+              <li><a href="about.html">About</a></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <section class="home" id="home-bg">
+        <div class="dance-logo">
+          <img src="https://img.icons8.com/emoji/96/000000/dancing-girl.png" alt="Logo" />
+        </div>
+        <div class="title">
+          <h1>"The Imaginary Dance"</h1>
+          <h2>Bringing life through dance!</h2>
+        </div>
+        <div class="home-info">
+          <p>
+            “Consciousness expresses itself through creation. This world we live
+            in is the dance of the creator. Dancers come and go in the twinkling
+            of an eye but the dance lives on. On many an occasion when I am
+            dancing, I have felt touched by something sacred. In those moments,
+            I felt my spirit soar and become one with everything that exists.”
+          </p>
+        </div>
+        <div class="home-info-2">
+          <p>
+            “You dance love, and you dance joy, and you dance dreams. And I know if I can make you smile by jumping over a couple of couches or running through a rainstorm, then I’ll be very glad to be a song and dance man.”
+          </p>
+        </div>
+      </section>
```

Comment:
- [ ] The `header` element shouldn't be used to contain many types of text content it should be used instead as an introduction to a piece of content or section of a webpage., kindly update your `header` tag to a give more semantic meaning.
- [ ] The `hero` section (first section of the home page) should fill the whole screen view on `desktop` sized screens, the `font-size` is a bit smaller compared to the design. Kindly update the `hero` section to match the design.
- [ ] Content inside the `hero` section should be aligned a bit to the left on `desktop` screens.
Reference Design
![image](https://user-images.githubusercontent.com/33767951/142095151-85a9ec27-2b49-4fc1-a9a4-ca6374a2bf0c.png)
Your Styling
![image](https://user-images.githubusercontent.com/33767951/142337782-f08180eb-d563-4a38-bbbe-e64ed3039eb7.png)